### PR TITLE
doc: add minutes for 18 May 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-05-18/minutes.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+* Michael Dawson
+  * Charts Node version 8.  Some discussions on dedicated machine
+    being donated by Intel
+* Kunal
+  * looking into Acme air and memory footprint 
+* Kyle 
+  * just starting to get up to speed
+* Taylor
+  * just sitting in to find out what we.re talking about
```
Comment (kunalspathak) [inline meetings/2017-05-18/minutes.md]: typo: what we are talking

File: meetings/2017-05-18/minutes.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+* Jiapeng
+  * first time joining, UNB student, ramping up research on Node.js scalability.
+* Anil/Uttam i
```
Comment (kunalspathak) [inline meetings/2017-05-18/minutes.md]: typo: Remove extra `i`

File: meetings/2017-05-18/minutes.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Actions from last meeting
+* Michael and Gareth to work on additional benchmarks as discussed above
+  * This is ongoing background work going to drop from actions
+* Ali to open issue on ignition discussion 
+  * No longer really applicable, ignition is already coming in.  
+
+### Creating new jobs
+* some discussion on how to create new benchmark jobs
+* Michael.s been doing some work to enable the broader team to be able to manage
```
Comment (kunalspathak) [inline meetings/2017-05-18/minutes.md]: typo: Michael is been

File: meetings/2017-05-18/minutes.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  jenkins jobs. Michael will setup meeting to kick this off.
+
+### Node summit CFP
+* There was discussion around the group submitting either a talk or Panel.  
+* Anil volunteered to setup a meeting with the interested parties to flesh that out.
+
+### Questions on acme air numbers reported
+* Anil questions about acme air footprint, should we add in profile of memory
```
Comment (kunalspathak) [inline meetings/2017-05-18/minutes.md]: Should be `Kunal` questions

File: meetings/2017-05-18/minutes.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  usage as opposed to before/after.  Geometric mean ?  Kunal took action to
```
Comment (kunalspathak) [inline meetings/2017-05-18/minutes.md]: typo: Kunal to take action to submit PR to capture or add additional metric

**Comment by mhdawson** [comment]
landed as f0ceaae280c1cb0d9cf79943cab37973b13631f7

**Comment by mhdawson** [comment]
@kunalspathak thanks for the review/comments.

Pushed update with fixes and will land.

**Comment by mhdawson** [comment]
Landed as a2e69315ab02d809d47d8636963a64e5e4ecae94

**Pull request closed without merging**